packages/terra-clinical-result/src/flowsheet-result-cell/FlowsheetResultCell.jsx

```diff
@@ -266,6 +282,7 @@ const createFlowsheetResultCellDisplay = (resultDataSet, hideUnit, numericOverfl
     firstResultData,
     resultKeyID,
   } = unpackResultDataSet(resultDataSet);
+  const hasAccessoryIcons = (firstResultAttributes.comment || firstResultAttributes.modified || firstResultAttributes.unverified);
```

Comment:
Just curious, any reason to bump this line up here?

Reply:
It's being used in two different places so I pulled it out to re-use it:

1. https://github.com/cerner/terra-clinical/blob/b063110b15c2ad1f1345416f80380b51063bfe75/packages/terra-clinical-result/src/flowsheet-result-cell/FlowsheetResultCell.jsx#LL326C7-L326C7
2. https://github.com/cerner/terra-clinical/blob/b063110b15c2ad1f1345416f80380b51063bfe75/packages/terra-clinical-result/src/flowsheet-result-cell/FlowsheetResultCell.jsx#L353